oauth2/src/main/java/com/okta/spring/boot/oauth/config/OktaOAuth2Properties.java

```diff
@@ -71,6 +71,11 @@
      */
     private String postLogoutRedirectUri;
 
+    /**
+     * Identifies Opaque token
+     */
+    private Boolean isOpaque;
```

Comment:
```suggestion
    private boolean opaque;
```

Use a simple boolean (only deal with two states) when possible

Reply:
don't prefix a bool with `is`